src/components/partnership/partnership.service.ts

```diff
@@ -383,21 +383,82 @@ export class PartnershipService {
       throw new NotFoundException('could not find Partnership');
     }
 
+    const projectMapPartner = this.db
+      .query()
+      .match(matchSession(session, { withAclRead: 'canReadPartnerships' }))
+      .match([node('partnership', 'Partnership', { active: true, id })]);
+    projectMapPartner.optionalMatch([
+      node('requestingUser'),
+      relation('in', '', 'member', { active: true }),
+      node('', 'SecurityGroup', { active: true }),
+      relation('out', '', 'permission', { active: true }),
+      node('projectMouStartPerm', 'Permission', {
+        property: 'mouStart',
+        active: true,
+        read: true,
+      }),
+      relation('out', '', 'baseNode', { active: true }),
+      node('project'),
+      relation('out', '', 'mouStart'),
+      node('projectMouStart', 'Property', {
+        active: true,
+      }),
+    ]);
+    projectMapPartner.optionalMatch([
+      node('requestingUser'),
+      relation('in', '', 'member', { active: true }),
+      node('', 'SecurityGroup', { active: true }),
+      relation('out', '', 'permission', { active: true }),
+      node('projectMouEndPerm', 'Permission', {
+        property: 'mouEnd',
+        active: true,
+        read: true,
+      }),
+      relation('out', '', 'baseNode', { active: true }),
+      node('project'),
+      relation('out', '', 'mouEnd'),
+      node('projectMouEnd', 'Property', {
+        active: true,
+      }),
+    ]);
+
+    projectMapPartner.return({
+      projectMouStart: [{ value: 'projectMouStart' }],
+      projectMouStartPerm: [
+        {
+          read: 'canReadProjectMouStart',
+          edit: 'canEditProjectMouStart',
+        },
+      ],
+      projectMouEnd: [{ value: 'projectMouEnd' }],
+      projectMouEndPerm: [
+        {
+          read: 'canReadProjectMouEnd',
+          edit: 'canEditProjectMouEnd',
+        },
+      ],
+    });
```

Comment:
It shouldn't be necessary to write new logic when the project service has a readOne function. Please refactor this to call the project service readOne function to get the project mouStart/End dates with permissions.

Reply:
@michaelmarshall , I need to find project(s) mapped with this partnership.